package.json

```diff
@@ -23,21 +23,21 @@
     "url": "https://github.com/ChenLi0830/dynamoLight"
   },
   "dependencies": {
-    "aws-sdk": "^2.504.0"
+    "aws-sdk": "^2.1281.0"
   },
   "devDependencies": {
-    "@types/jest": "^24.0.17",
-    "@types/node": "^12.6.9",
-    "big.js": "^5.2.2",
-    "chai": "^4.2.0",
-    "jest": "^24.8.0",
-    "prettier": "^1.18.2",
-    "serverless": "^1.49.0",
-    "serverless-dynamodb-local": "^0.2.38",
-    "ts-jest": "^24.0.2",
-    "tslint": "^5.18.0",
+    "@types/jest": "^29.2.4",
+    "@types/node": "^18.11.17",
+    "big.js": "^6.2.1",
+    "chai": "^4.3.7",
+    "jest": "^29.3.1",
+    "prettier": "^2.8.1",
+    "serverless": "^3.26.0",
```

Comment:
The `serverless` upgrade broke testing framework. `serverless-dynamodb-local` doesn't work well with serverless 3.0.